README.md

```diff
@@ -10,6 +10,13 @@ examples get run and tailor the output.
     gem install rspec-core # for rspec-core only
     rspec --help
 
+  Want to run against the `master` branch? You'll need to include the dependent
+  RSpec repos as well, add the following to your `Gemfile`:
+
+    %w[rspec rspec-expectations rspec-mocks rspec-support].each do |lib|
```

Comment:
I pulled this directly from the `Gemfile`. I was thinking "in-addition to this repo". But I guess, you're correct - it should just be all the repos.